tools/Install/setup.sh

```diff
@@ -246,41 +253,48 @@ echo
+STARTUP_SCRIPT=$CURRENT_DIR/.avsrun-startup.sh
 if [ ! -f $AUTOSTART ]; then
     mkdir -p $AUTOSTART_DIR
     cp /etc/xdg/lxsession/LXDE-pi/autostart $AUTOSTART
 fi
-STARTUP_SCRIPT=$CURRENT_DIR/.avsrun-startup.sh
 cat << EOF > "$STARTUP_SCRIPT"
 #!/bin/bash
 $AVSRUN_CMD
-\$SHELL
 EOF
+
+#if vocalfusion_3510_sales_demo is present, modify .avsrun-startup.sh to start sales demo along with AVS
+if [ -f $VOCALFUSION_3510_SALES_DEMO_PATH_FILE ]; then
+    SALES_DEMO_PATH=`cat $VOCALFUSION_3510_SALES_DEMO_PATH_FILE`
+    sed -i '/'"offline_demo"'/d' $STARTUP_SCRIPT
+    echo "lxterminal -t offline_demo -e \"$SALES_DEMO_PATH/"run_demo.sh" --with-avs\" &" >> $STARTUP_SCRIPT
+fi
+
 chmod a+rx $STARTUP_SCRIPT
+
 while true; do
     read -p "Automatically run AVS SDK at startup (y/n)? " ANSWER
     case ${ANSWER} in
         n|N|no|NO )
-            if grep $AUTOSTART_SESSION $AUTOSTART; then
+            if grep $AUTOSTART_SESSION $AUTOSTART; then 
                 # Remove startup script from autostart file
                 sed -i '/'"$AUTOSTART_SESSION"'/d' $AUTOSTART
             fi
             break;;
         y|Y|yes|YES )
-            if ! grep $AUTOSTART_SESSION $AUTOSTART; then #avsrun not present
-                if ! grep "vocalfusion_3510_sales_demo" $AUTOSTART; then #vocalfusion_3510_sales_demo not present
-                    # Append startup script if not already in autostart file
-                    echo "@lxterminal -t $AUTOSTART_SESSION --geometry=150x50 -e $STARTUP_SCRIPT" >> $AUTOSTART
-                fi
-            else #avsrun present
-                if grep "vocalfusion_3510_sales_demo" $AUTOSTART ; then #vocalfusion_3510_sales_demo present
-                    # Remove startup script from autostart file
-                    echo "Warning: Not adding avsrun in autostart since offline demo is already present. Start AVS by following instructions on vocalfusion_3510_sales_demo startup"
-                    sed -i '/'"$AUTOSTART_SESSION"'/d' $AUTOSTART
-                fi
-            fi
+	    if [ ! -f $VOCALFUSION_3510_SALES_DEMO_PATH_FILE ]; then #sales demo is not present
+	        #remove existing lines containing avs
+	        sed -i '/'"$AUTOSTART_SESSION"'/d' $AUTOSTART
+	        # Append avs startup script in autostart file
+	        echo "@$STARTUP_SCRIPT" >> $AUTOSTART
+	    else #vocalfusion demo present and we're adding avs so update the startup.py call in $AUTOSTART
+    		SALES_DEMO_PATH=`cat $VOCALFUSION_3510_SALES_DEMO_PATH_FILE`
+		sales_demo_startup_string="@lxterminal -t startup -e python3 $SALES_DEMO_PATH/startup.py --avsrun-startup-file $STARTUP_SCRIPT"
+		sed -i '/'"vocalfusion_3510_sales_demo"'/d' $AUTOSTART
+	        sed -i '/'"$AUTOSTART_SESSION"'/d' $AUTOSTART
+		echo "$sales_demo_startup_string" >> $AUTOSTART
+	    fi
```

Comment:
It looks like tabs have been added here. If this is case, can we replace them with spaces?